lib/screens/second_screen.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:dio/dio.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_localization/data/album.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+class SecondScreen extends StatelessWidget {
+  const SecondScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(AppLocalizations.of(context)!.secondScreen),
+      ),
+      body: FutureBuilder<List<dynamic>?>(
+          future: getHttp(),
+          builder: (context, snapshot) {
+            Widget child = const Center(
+              child: CircularProgressIndicator(),
+            );
+            if (snapshot.connectionState == ConnectionState.done) {
+              if (snapshot.hasError) {
+                child = Center(
+                  child: Text(
+                    AppLocalizations.of(context)!.weHaveAnError,
+                    style: const TextStyle(
+                      fontSize: 16,
+                    ),
+                  ),
+                );
+              } else if (snapshot.hasData) {
+                child = ListAlbums(list: snapshot);
+              }
+            }
+            return child;
+          }),
+    );
+  }
+
+  Future<List<Album>?> getHttp() async {
```

Comment:
Please create AlbumRepository with private filed `Dio dioClient;` and method `getAlbums` and move this code there. Widgets should not contain network logic. Then you will call `albumRepository.getAlbums()` in line 16